README.md

```diff
@@ -1,180 +1,95 @@
-
-
-**Qiskit Metal** is an open-source framework for designing quantum devices.
-
-
+# Qiskit Metal 
+[![License](https://img.shields.io/github/license/Qiskit/qiskit-metal.svg?style=popout-square)](https://opensource.org/licenses/Apache-2.0)<!--- long-description-skip-begin -->[![Release](https://img.shields.io/github/release/Qiskit/qiskit-metal.svg?style=popout-square)](https://github.com/Qiskit/qiskit-metal/releases)<!--- long-description-skip-begin -->[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.4618153.svg)](https://doi.org/10.5281/zenodo.4618153)
 >  Quantum hardware design and analysis
-
 ![Welcome to Qiskit Metal!](docs/images/zkm_banner.png 'Welcome to Qiskit Metal')
-
-### Quantum device design project
-We are thrilled to ask you to join this journey to revolutionize quantum devices. This is a first-of-its-kind, open-source project for engineers and scientists to design superconducting quantum devices with ease.
-
-Qiskit Metal is licensed under Apache 2.0. IBM reserves no copyright over outputs of Qiskit Metal.
-
-#### Watch the recorded tutorials
-The streaming will also be recorded and made available [here](https://www.youtube.com/playlist?list=PLOFEBzvs-VvqHl5ZqVmhB_FcSqmLufsjb) for offline review.
-
-#### Take part in the live tutorials and discussion
-Through June 2021 we are offering live tutorials and Q&A. [Sign up](https://airtable.com/shrxQEgKqZCf319F3) to receive an invite to the upcoming sessions.  The streaming will also be recorded and made available for offline review.  Find [here](https://github.com/Qiskit/qiskit-metal/blob/main/README_Tutorials.md) more details on schedule and use the Slack channel to give us feedback and to request the most relevant content to you.
-
-#### Get help: Slack
-Use the slack channel.  Join [qiskit slack](https://ibm.co/joinqiskitslack) and then join the `#metal` channel to communicate with the developers and other participants.  You may also use this channel to inquire about collaborations.
-
-## Documentation
-Full code documentation can be found [here](https://qiskit.org/documentation/metal/)
-
-You can open the page also by executing:
-```python
-import qiskit_metal
-qiskit_metal.open_docs()
+Qiskit is an open-source framework for working with noisy quantum computers at the level of pulses, circuits, and algorithms.
+While much of Qiskit is made up of elements that each work together to enable quantum computing, this element, Metal, is an open-source framework for engineers and scientists to design superconducting quantum devices with ease.
+## Installation
+We encourage installing Qiskit Metal via the pip tool (a python package manager).
+```bash
+pip install qiskit-metal
 ```
-
-Finally, you can also choose to build the documentation locally. To do so, from command line, change directory to `docs` and execute `python build_docs.py`.
-
-## Installation from the github repository
```

Comment:
it seems the github-install instructions are removed. Can we instead move them to a separate `README_github.md`, or `README_contributor.md` or something like it? And maybe add a note here that is not much that we "encourage" them to install t this way, but more neutral: `to just use the tool, do this PyPI install`, and then add that `to edit the code and truly customize the experience, or if PyPI gives trouble, do the github install`......something like that.

Reply:
Yes, the question, "what about developers" jumped out immediately at me.
- In the current readme there is a reference do `requirements-dev.txt`, which contains things like `pylint`, `yapf` among other things that developers will need to set up their environment.
- We also seem to have lost the `NEW_DEVELOPER_SETUP.md` instructions.

Reply:
Done!